orc8r/cloud/go/module.mk

```diff
@@ -29,6 +29,9 @@ fmt::
 	gofmt -s -w .
 
 gen::
+	# obsidian/swagger/gen.go depends on obsidian/swagger/protos/gen.go, so it
+	# must be generated first.
+	go generate obsidian/swagger/protos
```

Comment:
@hcgatewood after orc8r/cloud/go/obsidian/swagger/gen.go was added, `make fullgen` in `orc8r/cloud` started failing because `magma/orc8r/cloud/go/obsidian/swagger` depends on generated .pb.go files from `magma/orc8r/cloud/go/obsidian/swagger/protos` (and I think `go generate ./...` runs in lexicographic order).

Not sure if we've run into this anywhere else before; let me know if there's a better way to get this dependency sorted properly. Note: if preferred, I think I can change line 35 to just have this package listed before `./...` with the same comment.

Reply:
oops, I had to move this to [Makefile](https://github.com/magma/magma/commit/5b363f0ca16d83e3fbdd98be0c31087f52b381fd#diff-6407cdf0d027d391dd9c70c218d9ee3a8728895ac8e6029e41906afa5a81b809R13) since module.mk is shared.